app/forms/disk-create.tsx

```diff
@@ -76,7 +76,7 @@ export function CreateDiskSideModalForm({
   const createDisk = useApiMutation('diskCreate', {
     onSuccess(data) {
       queryClient.invalidateQueries('diskList')
-      addToast({ content: 'Your disk has been created' })
+      addToast({ content: `${data.name} created` })
```

Comment:
These names might benefit from something like the `HL` treatment we currently use in some modals.

<img width="542" alt="image" src="https://github.com/user-attachments/assets/3f6c5c25-1520-4e64-b02c-d0ef7c703527">


https://github.com/oxidecomputer/console/blob/a76ba5f46f2509d21c56d397e1f1cf62bd8dbbbb/app/pages/project/instances/instance/tabs/NetworkingTab.tsx#L349-L362